test/e2e_node/critical_pod_test.go

```diff
@@ -58,8 +56,11 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
 				framework.Skipf("unable to run test without dynamic kubelet config enabled.")
 			}
 			// because adminssion Priority enable, If the priority class is not found, the Pod is rejected.
-			systemCriticalPriority := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: systemCriticalPriorityName}, Value: scheduling.SystemCriticalPriority}
-
+			for _, pc := range schedulingapiv1.SystemPriorityClasses() {
+				_, err = f.ClientSet.SchedulingV1().PriorityClasses().Get(pc.GetName(), metav1.GetOptions{})
+				framework.ExpectNoError(err, "failed to get PriorityClasses %s with an error: %v", pc.GetName(), err)
+			}
```

Comment:
I would drop this loop/check. The only one we actually care about is SystemNodeCritical, and we will get a specific error if that is not present when we try to create the pod.